src/mempool.rs

```diff
@@ -1201,6 +1243,14 @@ pub mod test {
         }
     }
 
+    pub fn create_data_vote(
+        crypto: &BlstCrypto,
+        hash: DataProposalHash,
+        size: LaneBytesSize,
+    ) -> Result<MempoolNetMessage> {
+        Ok(MempoolNetMessage::DataVote(crypto.sign((hash, size))?))
+    }
```

Comment:
not convinced by this new function, but fine

Reply:
or maybe as `self.create_data_vote(hash, size)`

Reply:
(c'est pour les tests fyi)